src/util/file.rs

```diff
@@ -18,10 +18,7 @@ use std::path::Path;
 use crc::crc32::{self, Digest, Hasher32};
 
 pub fn read_all<P: AsRef<Path>>(path: P) -> io::Result<Vec<u8>> {
```

Comment:
Seems the method can be removed.

Reply:
Turns out this is only used in `src/util/security.rs`. I'll replace it.